### PR TITLE
align headers, clean up imports & fix typo

### PR DESCRIPTION
**Pull request by MatthiasWinzeler**
Hi folks

As discussed with @rainerleber in https://github.com/sap-linuxlab/community.sap_launchpad/pull/16#discussion_r1386496873, the file headers are in need of some alignment ;)

This PR does the following:

- remove any encoding header since [they are not needed with python 3](https://stackoverflow.com/a/56118631) (do we need to support python 2 still?)
- remove any shebangs in the module_utils, add them to modules (only these are invoked directly)

Along the way I fixed a typo and arranged some imports - see inline comments.

### CODE REVIEW

File: plugins/module_utils/constants.py [hunk 1]
```diff
@@ -6,8 +6,8 @@
 URL_SERVICE_INCIDENT = 'https://launchpad.support.sap.com/services/odata/incidentws'
 URL_SERVICE_USER_ADMIN = 'https://launchpad.support.sap.com/services/odata/useradminsrv'
 URL_SOFTWARE_DOWNLOAD = 'https://softwaredownloads.sap.com'
-# Maintainance Planner
-URL_MAINTAINANCE_PLANNER = 'https://maintenanceplanner.cfapps.eu10.hana.ondemand.com'
+# Maintenance Planner
```
Comment (MatthiasWinzeler) [inline plugins/module_utils/constants.py:9]: fixed typo

File: plugins/module_utils/sap_launchpad_maintenance_planner_runner.py [hunk 2]
```diff
@@ -11,11 +7,10 @@
 from bs4 import BeautifulSoup
 from lxml import etree
 from requests.auth import HTTPBasicAuth
-from requests.sessions import session
 
 from . import constants as C
 from .sap_api_common import _request, https_session
-from .sap_id_sso import _get_sso_endpoint_meta, sap_sso_login
+from .sap_id_sso import _get_sso_endpoint_meta
```
Comment (MatthiasWinzeler) [inline plugins/module_utils/sap_launchpad_maintenance_planner_runner.py:13]: there was no reference to `sap_sso_login` in this file, causing linters to complain - so I moved the imports into the files that actually reference the `sap_sso_login` function.

File: plugins/module_utils/sap_launchpad_software_center_catalog_runner.py [hunk 3]
```diff
@@ -1,9 +1,5 @@
-#!/user/bin/env python3
-# coding: utf-8
-
 from . import constants as C
 from .sap_api_common import _request
-from .sap_id_sso import sap_sso_login
```
Comment (MatthiasWinzeler) [inline plugins/module_utils/sap_launchpad_software_center_catalog_runner.py:6]: see https://github.com/sap-linuxlab/community.sap_launchpad/pull/17/files#r1397301909

File: plugins/module_utils/sap_launchpad_software_center_download_runner.py [hunk 4]
```diff
@@ -12,7 +9,7 @@
 
 from . import constants as C
 from .sap_api_common import _request, https_session
-from .sap_id_sso import _get_sso_endpoint_meta, sap_sso_login
+from .sap_id_sso import _get_sso_endpoint_meta
```
Comment (MatthiasWinzeler) [inline plugins/module_utils/sap_launchpad_software_center_download_runner.py:12]: see https://github.com/sap-linuxlab/community.sap_launchpad/pull/17/files#r1397301909

File: plugins/module_utils/sap_launchpad_software_center_download_search_fuzzy.py [hunk 5]
```diff
@@ -5,7 +5,6 @@
 
 from . import constants as C
 from .sap_api_common import _request
-from .sap_id_sso import sap_sso_login
```
Comment (MatthiasWinzeler) [inline plugins/module_utils/sap_launchpad_software_center_download_search_fuzzy.py:8]: see https://github.com/sap-linuxlab/community.sap_launchpad/pull/17/files#r1397301909

File: plugins/modules/maintenance_planner_files.py [hunk 6]
```diff
@@ -69,6 +69,7 @@
 from ..module_utils.sap_launchpad_maintenance_planner_runner import *
 from ..module_utils.sap_launchpad_software_center_download_runner import \
     is_download_link_available
+from ..module_utils.sap_id_sso import sap_sso_login
```
Comment (MatthiasWinzeler) [inline plugins/modules/maintenance_planner_files.py:72]: https://github.com/sap-linuxlab/community.sap_launchpad/pull/17/files#r1397301909

File: plugins/modules/maintenance_planner_stack_xml_download.py [hunk 7]
```diff
@@ -69,6 +69,7 @@
 
 # Import runner
 from ..module_utils.sap_launchpad_maintenance_planner_runner import *
+from ..module_utils.sap_id_sso import sap_sso_login
```
Comment (MatthiasWinzeler) [inline plugins/modules/maintenance_planner_stack_xml_download.py:72]: https://github.com/sap-linuxlab/community.sap_launchpad/pull/17/files#r1397301909

File: plugins/modules/software_center_download.py [hunk 8]
```diff
@@ -88,6 +88,7 @@
 
 # Import runner
 from ..module_utils.sap_launchpad_software_center_download_runner import *
+from ..module_utils.sap_id_sso import sap_sso_login
```
Comment (MatthiasWinzeler) [inline plugins/modules/software_center_download.py:91]: https://github.com/sap-linuxlab/community.sap_launchpad/pull/17/files#r1397301909

**Review by rainerleber — APPROVED** [review]
LGTM, @MatthiasWinzeler thank you for the alignment :-)

**Comment by MatthiasWinzeler** [comment]
@rainerleber thanks! Can you merge the PR? I'm afraid I don't have the rights:

> Only those with [write access](https://docs.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests.

**Pull request merged**